plotextractor/extractor.py

```diff
@@ -198,6 +198,7 @@ def extract_captions(tex_file, sdir, image_list, primary=True):
     subfloat_head = u'\\subfloat'
     subfig_head = u'\\subfigure'
     includegraphics_head = u'\\includegraphics'
+    include_head = r'^\\include{'
```

Comment:
is it always the case that `{` is after include? Maybe it would be safer to add negative lookbehind?

Reply:
Yes, `{` is always there. Okay, I can add this.